version: '3.4'
services:
  plex:
    image: linuxserver/plex
    container_name: plex
    network_mode: host
    environment:
      - PUID=1000
      - PGID=1000
      - VERSION=docker
      - PLEX_CLAIM= #optional
    volumes:
      - ${PLEX_CONFIG_PATH}:/config
      - /path/to/tvseries:/tv
      - /path/to/movies:/movies
    ports:
      - ${PLEX_EXTERNAL_PORT}:${PLEX_INTERNAL_PORT}
    healthcheck:
        test: ["CMD", "curl", "-f", "http://localhost:${PLEX_INTERNAL_PORT}"]
        interval: 10s
        timeout: 10s
        retries: 5
        start_period: 30s
    restart: unless-stopped

  plexwolfromlog:
    container_name: plexwolfromlog
    # image: #plexwolfromlog
    environment:
      - LOG_FILE_NAME=Plex Media Server.log
      - PLEX_PATH=${PLEX_CONFIG_PATH}
      - PART_OF_LOG_PATH=Library/Application Support/Plex Media Server/Logs/
      - MAC_ADDRESS=18:C0:4D:34:63:A8
      - IP_ADDRES=192.168.0.11
    depends_on:
      plex:
        condition: service_healthy
    links:
        - plex
    build:
      context: .
      dockerfile: ./Dockerfile
    restart: on-failure